crux-bench/time-series-bench/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM clojure:openjdk-11-lein-2.9.0
+
+VOLUME /root/.m2
+
+WORKDIR /time-series-bench
+
+CMD ["lein", "run"]
+
+ADD project.clj .
+ADD src/ ./src/
+
+ADD https://timescaledata.blob.core.windows.net/datasets/devices_small.tar.gz ./resources/
```

Comment:
I thought the `ADD` command decompressed these automatically, evidently it doesn't for remote files. let's combine this with your download_dataset script - i.e. assume these have already been downloaded to resources

Reply:
Haha - so did we. Seems sensible, stops it from downloading each time :+1: